schema.graphql

```diff
@@ -924,6 +934,12 @@ type Provision @entity {
   delegatorShares: BigInt!
   "Exchange rate of of tokens received for each share"
   delegationExchangeRate: BigDecimal!
+  "Ratio between the amount of the indexers own stake over the total usable stake."
+  ownStakeRatio: BigDecimal!
+  "Ratio between the amount of delegated stake over the total usable stake."
+  delegatedStakeRatio: BigDecimal!
+  "Percentage of indexers' own rewards received in relation to its own stake. 1 (100%) means that the indexer is receiving the exact amount that is generated by his own stake"
+  indexerRewardsOwnGenerationRatio: BigDecimal!
```

Comment:
Maybe worth clarifying this ratio can be below or above 1 depending on the delegation fee cuts.

Reply:
To be honest, all these advanced metrics were implemented really back in the day when we still used spreadsheets to figure out potentially interesting metrics that could tell us how indexers behaved on a protocol that only had indexing rewards.

I doubt people are using them any more, but even so, we might want to eventually remove the advanced metrics and keep things simple, as the approach of keep adding more and more advanced stuff is what lead us to this state where we have a huge amount of fields that all/most people won't ever use and that require porting or maintenance every so often (during upgrades for example)